docs/docs/debugging.md

```diff
@@ -2,6 +2,6 @@
 title: Debugging
 ---
 
-While building your site, you’ll sometimes encounter bugs. These guides help you set up debugging in Gatsby so you can spot and squash bugs more easily.
+While building your site, you’ll sometimes encounter bugs. These guides help you set up debugging in Gatsby so you can spot and squash bugs more.
```

Comment:
While the occurrence may be valid (not completely sure), removing the word like this leaves an awkward sentence at best, if not actually an incorrect sentence. Something like "so it makes it easier for you to spot bugs", or something in that line.